modules/aws_asg/check-docker-health.bash

```diff
@@ -45,32 +44,37 @@ main() {
 
 __handle_implode_confirm() {
   local run_d="${1}"
-  local post_sleep="${2}"
+  local pre_implosion_sleep="${2}"
 
   local reason
   reason="$(cat "${run_d}/implode.confirm" 2>/dev/null)"
   : "${reason:=not sure why}"
   "${SHUTDOWN:-/sbin/shutdown}" -P now "imploding because ${reason}"
-  sleep "${post_sleep}"
 }
```

Comment:
I see the `pre_implosion_sleep` local var declared here, but used (undeclared) in `__handle_unresponsive_docker` 😕 

IIRC the point of the previous `sleep "${post_sleep}"` was to allow time for `shutdown` to take effect without re-running all these checks immediately, which made sense in the context of `travis-worker` hook scripts, but I can see how it's not necessary here 👍

Reply:
Good catches :/